3rd/src/031Exception.py

```diff
@@ -0,0 +1,17 @@
+#! python3
+
+#エラーが発生した際に別処理へ遷移させたい箇所をtry～exceptで囲む。
+#基本的に全部でいい。
+try:
+
+#例文として存在しないファイルを指定
+    output = open("exception.txt", "r",encoding="utf-8")
+
+    for o in output:
+        print (o);
+        output.close()

```

Comment:
```suggestion
    output.close()
````
`output.close()`がforの中に入ってしまっているため、出したほうがいいと思います